src/content/docs/ai-monitoring/view-ai-data.mdx

```diff
@@ -0,0 +1,187 @@
+
+If you own several apps with various implementations of different AI frameworks, you can get a general sense for how your AI models perform.  
+
+### Track total responses, average response time, and token usage  
+
+<img
+    title="AI responses response billboard and timeseries graphs"
+    alt="A cropped screenshot displaying the timeseries graphs and billboard info about AI data"
+    src={aiTimeseriesBillboard}
+/>
+
+The three tiles show general performance metrics about your AI's responses. These tiles may not tell you the exact cause behind a problem, but they're useful for identifying anomalous fluctuations in your app's performance. 
+
+<img
+    title="Three tiles to show spikes and drops in response usage"
+    alt="A cropped screenshot displaying billboard info about AI data"
+    src={aiCroppedImageofAIBillboards}
+/>
+
+* If you notice a drop in total responses or increase in average response time, it can indicate that some technology in your AI toolchain has malfunctioned and prevented your AI-powered app from posting a response.  
+* A drop or increase in average token usage per response can give you insight into how your model creates a response. Maybe it's pulling too much context, thus driving up token cost while generating its response. Maybe its responses are too spare, leading to lower token costs and unhelpful responses.
+
+### Adjust the timeseries graphs 
+
+<img
+    title="AI timeseries graphs"
+    alt="A cropped screenshot displaying timeseries info about AI data"
+    src={aiCroppedImageofAItimeseries}
+/>
+
+You can refer to the time series graphs to better visualize when an anamolous behavior first appears. 
+
+* Adjust the timeseries graph by dragging over a spike or drop. This scopes the timeseries to a certain time window.
+* Select the drop down to run comparative analysis for different performance parameters. You can choose between total responses, average response time, or average tokens per response. 
+* If you've enabled the feedback feature, you can scope the graphs to analyze responses by positive and negative feedback. 
+
+### Evaluate individual AI responses 
+
+Your AI response table organizes data about interactions between your end user and AI app. You can view when an interaction occurred, prompts paired with their responses, completion and token count, and which model received a prompt. 
+
+<img
+    title="AI response page response table"
+    alt="A cropped screenshot displaying the response table from the AI responses view"
+    src={aiResponseTable}
+/>
+
+You can adjust the table columns by clicking the cog icon in the upper right. This functionality lets you choose the kinds of data you want to analyze.
+
+The Responses is an entry point into viewing trace data about an individual response. Click a row in the table to open the trace view of a particular response.
```

Comment:
I think that using our writing style, "Responses" is correct, but "The Responses is" reads VERY weirdly in the way it's presented.